src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.model.exception.RegistrationException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User validUser = new User("Mitresko", "P@ssw0rd", 30);
+    private static final User invalidUserLoginExist = new User("Mitresko", "P@ssw0rd", 30);
+    private static final User invalidUserAgeTooYoung = new User("Mitresko1", "P@ssw0rd", 17);
+    private static final User invalidUserShortPass = new User("Taras", "P@ssw", 30);
+    private static final User invalidUserNullLogin = new User(null, "P@ssw", 30);
+    private static final User invalidUserNullPass = new User("Mykola", null, 30);
+    private static final User invalidUserNullAge = new User("Sashko", "P@ssword", null);
+    private static RegistrationService service;
+
+    @BeforeEach
+    void clearData() {
+        Storage.people.clear();
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void addAndGet_validUser_Ok() {
```

Comment:
```suggestion
    public void register_validUser_Ok() {
```
remember about access modifiers, you are testing register method